packages/patternfly-4/react-core/src/demos/FilterTableDemo.md

```diff
@@ -0,0 +1,388 @@
+      <SelectOption key="statusMaint" value="Needs Maintainence" />
+    ];
+
+    return (
+      <React.Fragment>
+        <DataToolbarGroup variant="filter-group" style={{ width: '55%' }}>
+          <DataToolbarFilter
+            chips={filters.location}
+            deleteChip={this.onDelete}
+            categoryName="Location"
+            showToolbarItem={currentCategory === 'Location'}
+            style={{ width: '100%' }}
+          >
+            <Select
+              aria-label="Location"
+              onToggle={this.onFilterToggle}
+              onSelect={this.onLocationSelect}
+              selections={filters.location[0]}
+              isExpanded={isFilterDropdownOpen}
+              placeholderText="Any"
+            >
+              {locationMenuItems}
+            </Select>
+          </DataToolbarFilter>
+          <DataToolbarFilter
+            chips={filters.name}
+            deleteChip={this.onDelete}
+            categoryName="Name"
+            showToolbarItem={currentCategory === 'Name'}
+            style={{ width: '100%' }}
+          >
+            <InputGroup>
+              <TextInput
+                name="nameInput"
+                id="nameInput1"
+                type="search"
+                aria-label="name filter"
+                onChange={this.onInputChange}
+                value={inputValue}
+                placeholder="Filter by name..."
+                onKeyDown={this.onNameInput}
+              />
+              <Button
+                variant={ButtonVariant.control}
+                aria-label="search button for search input"
+                onClick={this.onNameInput}
+              >
+                <SearchIcon />
+              </Button>
+            </InputGroup>
+          </DataToolbarFilter>
+          <DataToolbarFilter
+            chips={filters.status}
+            deleteChip={this.onDelete}
+            categoryName="Status"
+            showToolbarItem={currentCategory === 'Status'}
+            style={{ width: '100%' }}
+          >
+            <Select
+              variant={SelectVariant.checkbox}
+              aria-label="Status"
+              onToggle={this.onFilterToggle}
+              onSelect={this.onStatusSelect}
+              selections={filters.status}
+              isExpanded={isFilterDropdownOpen}
+              placeholderText="Filter by status"
+            >
+              {statusMenuItems}
+            </Select>
+          </DataToolbarFilter>
+        </DataToolbarGroup>
+      </React.Fragment>
+    );
+  }
+
+  renderToolbar() {
+    const { filters } = this.state;
+    return (
+      <DataToolbar
+        id="data-toolbar-with-chip-groups"
+        clearAllFilters={this.onDelete}
+        showClearFiltersButton={
+          filters.location.length !== 0 || filters.name.length !== 0 || filters.status.length !== 0
+        }
+      >
+        <DataToolbarContent>
+          <DataToolbarGroup style={{ width: '50%' }}>
```

Comment:
If we need to make this toolbar responsive, you should only need to use a DataToolbarToggleGroup rather than a DataToolbarGroup here. You should at least try it (whether or not it ends up in the demo) to make sure that your multi-drop-down style filter still works in a responsive toggle group.

Reply:
feel free to reach out to me if it doesn't. I think the changes you made to the DataToolbarFilter component are good. So it'll just come down to how you've implemented it in the demo to get it to work.